site/content/contribute/good-decisions/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Section 2 — Making good decisions"
+heading: "Section 2 — Making good decisions"
+description: ""
+date: "2022-10-01T12:00:00-00:00"
+weight: 2
+---
+## Subsection 1 — Be inclusive
+
+1. Hatred of any type is not welcome at Mattermost — harassment of any kind is grounds for immediate expulsion from this community.
+2. Bias can exist without hatred, so it’s always worth considering how to be more inclusive.
+    1. Depictions and representations of people should always be diverse in context. Where it’s possible, show people of different genders, races, and cultures. This also applies to choosing speakers at events.
+    2. Keep those with visual and auditory impairments in mind by using semantic and meaningful organization in documents, adding alternate text to images, and giving videos subtitles in American English.
+    3. Make sure not to accidentally discriminate by economic situation or physical location when making tooling, scheduling, and speaker management decisions.
+3. Everyone has the right to voice concerns about perceived discrimination, but be gentle even if the other person is not. Start with a kind private message to avoid embarrassment, but you can report the offender to whoever is in charge of the space in which the discrimination occurred. Be forgiving, especially if the person makes an effort to improve.
+
+## Subsection 2 — Work together
+
+1. Use established authorities to avoid and resolve debates. This guide contains numbered lines so that you can explicitly refer to the sections that show your opinion is in line with Mattermost’s principles.
+2. Always aim for clarity and conciseness. Plain, professional, and descriptive language solves many communication issues and avoids excluding those with learning disabilities, with different learning styles, and with less experience in this industry.
```

Comment:
@jordanafung - Deferring reviews of this content to you and your team.

Reply:
@neflyte - We're going to remove the specific guidelines listed below (#1 - #10) and replace it with a link to the Mattermost Contribution guidelines and code of conduct instead via https://handbook.mattermost.com/contributors/contributors/guidelines/contribution-guidelines